s3_management/manage.py

```diff
@@ -244,21 +236,6 @@ def __lt__(self, other):
         return self.key < other.key
 
 
-def extract_package_build_time(full_package_name: str) -> datetime:
-    result = search(PACKAGE_DATE_REGEX, full_package_name)
-    if result is not None:
-        with suppress(ValueError):
-            # Ignore any value errors since they probably shouldn't be hidden anyways
-            return datetime.strptime(result.group(2), "%Y%m%d")
-    return datetime.now()
-
-
-def between_bad_dates(package_build_time: datetime):
```

Comment:
This is no longer relevant since this was 3 years ago and we only keep the latest 60 builds.